source/Octopus.Tentacle.Kubernetes.Tests.Integration/KubernetesClusterOneTimeSetUp.cs

```diff
@@ -17,25 +17,32 @@ public async Task OneTimeSetUp()
         installer = new KubernetesClusterInstaller(KubernetesTestsGlobalContext.Instance.TemporaryDirectory, kindExePath, helmExePath, kubeCtlPath, KubernetesTestsGlobalContext.Instance.Logger);
         await installer.Install();
 
-        //if we are not running in TeamCity, then we need to find the latest local tag and use that if it exists 
-        if (!TeamCityDetection.IsRunningInTeamCity())
-        {
-            var imageLoader = new DockerImageLoader(KubernetesTestsGlobalContext.Instance.TemporaryDirectory, KubernetesTestsGlobalContext.Instance.Logger, kindExePath);
-            KubernetesTestsGlobalContext.Instance.TentacleImageAndTag = imageLoader.LoadMostRecentImageIntoKind(installer.ClusterName);
-        }
-        else
+        KubernetesTestsGlobalContext.Instance.TentacleImageAndTag = GetTentacleImageAndTag(kindExePath);
+        KubernetesTestsGlobalContext.Instance.SetToolExePaths(helmExePath, kubeCtlPath);
+        KubernetesTestsGlobalContext.Instance.KubeConfigPath = installer.KubeConfigPath;
+    }
+
+    string? GetTentacleImageAndTag(string kindExePath)
+    {
+        //By default, we don't override the values in the helm chart. This is useful if you are just writing new tests and not changing Tentacle code.
+        string? imageAndTag = null;
+        if (TeamCityDetection.IsRunningInTeamCity())
         {
+            //In TeamCity, use the tag of the currently building code
             var tag = Environment.GetEnvironmentVariable("KubernetesAgentTests_ImageTag");
-            KubernetesTestsGlobalContext.Instance.TentacleImageAndTag = $"docker.packages.octopushq.com/octopusdeploy/kubernetes-tentacle:{tag}";
+            imageAndTag = $"docker.packages.octopushq.com/octopusdeploy/kubernetes-tentacle:{tag}";
         }
-
-        if (KubernetesTestsGlobalContext.Instance.TentacleImageAndTag is not null)
+        else if(bool.TryParse(Environment.GetEnvironmentVariable("KubernetesAgentTests_UseLatestLocalImage"), out var useLocal) && useLocal)
```

Comment:
Now if you want to use a locally built image, you need to set the environment variable `KubernetesAgentTests_UseLatestLocalImage` to `true`